cli/polygonetl/jobs/export_traces_job.py

```diff
@@ -20,28 +20,32 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
-from polygonetl.executors.batch_work_executor import BatchWorkExecutor
+import sys
+
 from blockchainetl_common.jobs.base_job import BaseJob
+from polygonetl.executors.batch_work_executor import BatchWorkExecutor
 from polygonetl.mainnet_daofork_state_changes import DAOFORK_BLOCK_NUMBER
 from polygonetl.mappers.trace_mapper import EthTraceMapper
 from polygonetl.service.eth_special_trace_service import EthSpecialTraceService
-
 from polygonetl.service.trace_id_calculator import calculate_trace_ids
 from polygonetl.service.trace_status_calculator import calculate_trace_statuses
 from polygonetl.utils import validate_range
 
+sys.setrecursionlimit(1200)
```

Comment:
Two things come to mind:
1. Should we set this once in a central location? Since I noticed the same line is added to `cli/polygonetl/jobs/extract_geth_traces_job.py` as well. EDIT: I just noticed you went with a central definition first, and then changed it, so then I would just be curious if there is a specific reason.
2. A short comment explaining why this is necessary would be super useful, cause this is one of those things that would make me scratch my head if I come across this again in a few months (or weeks...).

Reply:
1. The idea was that I only want this to run when traces is enabled. but looking at it again, it'll still run regardless due to the import [here](https://github.com/blockchain-etl/polygon-etl/blob/main/cli/polygonetl/streaming/eth_streamer_adapter.py#L10). 
2. added comment